src/vivarium_public_health/risks/effect.py

```diff
@@ -260,24 +265,46 @@ def _rebin_relative_risk_data(
     def get_risk_exposure(self, builder: Builder) -> Callable[[pd.Index], pd.Series]:
         return builder.value.get_value(self.exposure_pipeline_name)
 
-    def get_target_modifier(
-        self, builder: Builder
-    ) -> Callable[[pd.Index, pd.Series], pd.Series]:
+    def adjust_target(self, index: pd.Index, target: pd.Series) -> pd.Series:
+        relative_risk = self.relative_risk_pipeline(index)
+        return target * relative_risk
+
+    def register_target_modifier(self, builder: Builder) -> None:
```

Comment:
Nit: if you move these register methods below `get_relative_risk`, they won't show up in the diff.